jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -247,21 +247,24 @@ if [ -z "$JETTY_HOME" ]; then
   exit 1
 fi
 
+RUN_DIR=$PWD
```

Comment:
Note that `$PWD` is a bash feature.
OSX is deprecating bash and moving to zsh. 
https://support.apple.com/en-us/HT208050
Basically, Apple is stuck with Bash 3.2 as it's GPLv2, but newer versions of bash are GPLv3.
Apple will not upgrade/ship and/or use anything with GPLv3 license.
They want to get rid of bash entirely simply as their version is very out of date, and have made it clear that it will go away in the future.

Perhaps we should move to `RUN_DIR=$(pwd)` instead? (it's more compatible)

Reply:
Just to be clear, there is zero urgency on this change.
Feel free to leave it as it is for now.

Reply:
ack